release/api/v1alpha1/artifacts.go

```diff
@@ -0,0 +1,143 @@
+package v1alpha1
+
+func (vb *VersionsBundle) Manifests() map[string][]Manifest {
+	manifests := map[string][]Manifest{}
```

Comment:
nit: this map is always the same, we could just initialize it with the proper values